docs/mobile-apps/features/device-vitals.md

```diff
@@ -0,0 +1,42 @@
+---
+id: device-vitals
+title: Device Vitals
```

Comment:
App Diagnostics

Reply:
As per our slack conversation- resolving the comment :)